### DISCUSSION TITLE
Trying to install on paper 1.17.1

### DISCUSSION THREAD

**Discussion by Julu2106** [eccentricdevotion/TARDIS#582] [Q&A]
I'm trying to install the plugin on a PaperMC 1.17.1 Server but whenever I try to restart the server the console says the TARDIS plugin can't be installed because it requires the ChunkGenerator even though the ChunkGenerator is right there in the plugins folder. Please, somebody help me... :(

**Comment by Chew** [comment]
Please ensure you're using the JARs from here: https://github.com/eccentricdevotion/TARDIS/releases/tag/1.17.1

Otherwise we need logs, versions, etc.

**Reply by Julu2106** [reply]
I took both files from that exact zip folder. But it won't work.

What exactly do you need from me?

**Reply by Julu2106** [reply]
That's my latest log
[latest.log](https://github.com/eccentricdevotion/TARDIS/files/7803842/latest.log)
:

**Reply by eccentricdevotion** [reply]
Attach your server log file - you can drag and drop the file into a comment here (preferred), or copy and paste from your server console.

**Comment by eccentricdevotion** [comment]
Looks like you need to update Paper

**Reply by eccentricdevotion** [reply]
You are 326 versions behind
https://papermc.io/downloads#Paper-1.17

**Reply by Julu2106** [reply]
How do I update that? Just pull it into the file manager?

**Reply by eccentricdevotion** [reply]
Maybe, your server host may have instructions?

**Reply by Julu2106** [reply]
The host gives a selection menu for the Gameversions and there's only one Version of Paper MC for 1.17.1

**Reply by eccentricdevotion** [reply]
Who is your host?

**Reply by eccentricdevotion** [reply]
Can you select a custom version?

**Reply by Julu2106** [reply]
Nitrado. Don't know if that's really a thing outside of Germany.

**Reply by Julu2106** [reply]
Found an option to upload my own version

**Reply by eccentricdevotion** [reply]
https://wiki.nitrado.net/en/Spigot#Manual_Installation

**Reply by Julu2106** [reply]
Version was changed succsesfully, the problem sadly remains the same.

**Reply by eccentricdevotion** [reply]
attach new log

**Reply by Julu2106** [reply]
[latest.log](https://github.com/eccentricdevotion/TARDIS/files/7803892/latest.log)

**Reply by eccentricdevotion** [reply]
Server needs to be running Java 16 or higher

**Reply by Julu2106** [reply]
I think it is. Is it not?

![Screenshot (9)](https://user-images.githubusercontent.com/97059811/147977383-c623d173-c772-413f-8048-96245766a73b.png)

**Reply by eccentricdevotion** [reply]
Try changing to 17

**Reply by Julu2106** [reply]
That did the job. Thanks so much for help and your patience!

**Reply by eccentricdevotion** [reply]
No problem, enjoy your TARDIS :)
